pegasus/sites.v3/code.org/public/css/teacher_info.scss

```diff
@@ -0,0 +1,60 @@
+@import "color.scss";
+@import "font.scss";
+
+.teacher-info-container {
+    background-color: $neutral_light;
```

Comment:
Our code generally uses 2 spaces for indentation, and presumably should for `.scss` files as well.

Reply:
Thanks for notice! Looks like a problem with my IDE, will fix